scripts/test.sh

```diff
@@ -11,7 +11,7 @@ fi
 
 package="community.flock.generated"
 
-languages=("Java" "Kotlin" "Scala" "TypeScript" "Wirespec")
+languages=("Java" "Kotlin" "TypeScript" "Python" "Wirespec" "OpenAPIV2" "OpenAPIV3")
```

Comment:
I see Scala tests removed, but not Python tests added.

Reply:
Python tests are added